src/core_atmosphere/physics/mpas_atmphys_init.F

```diff
@@ -308,7 +310,7 @@ subroutine physics_init(dminfo,stream_manager,clock,configs,mesh,diag,tend,state
  endif
 
 !initialization of xicem:
- if(.not.config_do_restart) then
+ if(.not.config_do_restart .or. .not.config_do_DAcycling) then
```

Comment:
@tanyasmirnova Should it be 
```
if(.not.config_do_restart .and. .not.config_do_DAcycling)
```
at Line 313 and 321?

Reply:
corrected